run_unittests.py

```diff
@@ -4632,7 +4639,10 @@ def main():
         # invokes our python wrapper
         batfile = os.path.join(self.builddir, 'binary_wrapper{}.bat'.format(self.current_wrapper))
         with open(batfile, 'wt') as f:
-            f.write('py -3 {} %*'.format(filename))
+            if mesonbuild.environment.detect_msys2_arch():
+                f.write(r"@%MSYS2_ROOT%\usr\bin\bash -c 'python3 {} %*'".format(filename.replace('\\', '/')))
```

Comment:
Depending on a CI only env var (MSYS2_ROOT) would make testing locally hard. Wouldn't "python3" alone work? It should be in PATH.

Reply:
Ah, so it does.  Thanks!